res/openvpn-gui-res-it.rc

```diff
@@ -563,33 +563,33 @@ Puoi far partire questa applicazione una volta che l'amministratore ha aggiornat
     /* AS profile import */
     IDS_ERR_URL_IMPORT_PROFILE "Errore durante il recupero del profilo dall'URL: [%d] %ls"
 
-    IDS_ERR_NO_PKCS11 "Nessun certificato trovato. Se hai un token inseriscilo e seleziona 'Riprova'."
+    IDS_ERR_NO_PKCS11 "Nessun certificato trovato.\nSe hai un token inseriscilo e seleziona 'Riprova'."
     IDS_ERR_SELECT_PKCS11 "Nessun certificato selezionato."
     /* column headers for pkcs11 certificate list */
     IDS_CERT_DISPLAYNAME "Rilasciato a"
     IDS_CERT_ISSUER "Rilasciato da"
     IDS_CERT_NOTAFTER "Valido fino a"
 
     /* PLAP related */
-    IDS_NFO_STATE_RETRYING "Retrying"
-    IDS_NFO_STATE_CANCELLING "Cancelling"
-    IDS_NFO_STATE_DISCONNECTING "Disconnecting"
-    IDS_NFO_CONN_CANCELLED "Connection cancelled by user"
+    IDS_NFO_STATE_RETRYING "Nuovo tentativo"
+    IDS_NFO_STATE_CANCELLING "Annullamento"
+    IDS_NFO_STATE_DISCONNECTING "Disconnessione"
+    IDS_NFO_CONN_CANCELLED "Connessione annullata dall'utente"
 
     /* openvpn daemon state names -- these are shown on progress dialog in PLAP */
-    IDS_NFO_OVPN_STATE_INITIAL      "Initializing"
-    IDS_NFO_OVPN_STATE_CONNECTING   "Connecting"
-    IDS_NFO_OVPN_STATE_ASSIGN_IP    "Assigning IP address"
-    IDS_NFO_OVPN_STATE_ADD_ROUTES   "Adding routes"
-    IDS_NFO_OVPN_STATE_CONNECTED    "Connected"
-    IDS_NFO_OVPN_STATE_RECONNECTING "Reconnecting"
-    IDS_NFO_OVPN_STATE_EXITING      "Exiting"
-    IDS_NFO_OVPN_STATE_WAIT         "Waiting"
-    IDS_NFO_OVPN_STATE_AUTH         "Authenticating"
-    IDS_NFO_OVPN_STATE_GET_CONFIG   "Pulling configuration from server"
-    IDS_NFO_OVPN_STATE_RESOLVE      "Resolving remote host"
-    IDS_NFO_OVPN_STATE_TCP_CONNECT  "Establishing TCP connection"
-    IDS_NFO_OVPN_STATE_AUTH_PENDING "Authentication pending"
+    IDS_NFO_OVPN_STATE_INITIAL      "Inizializzazione"
+    IDS_NFO_OVPN_STATE_CONNECTING   "Connessione"
+    IDS_NFO_OVPN_STATE_ASSIGN_IP    "Assegnazione indirizzo IP"
+    IDS_NFO_OVPN_STATE_ADD_ROUTES   "Aggiunta percorsi"
+    IDS_NFO_OVPN_STATE_CONNECTED    "Connesso"
+    IDS_NFO_OVPN_STATE_RECONNECTING "Riconnessione"
+    IDS_NFO_OVPN_STATE_EXITING      "Uscita"
+    IDS_NFO_OVPN_STATE_WAIT         "In attesa"
+    IDS_NFO_OVPN_STATE_AUTH         "Autenticazione"
+    IDS_NFO_OVPN_STATE_GET_CONFIG   "Recupero configurazione dal server"
+    IDS_NFO_OVPN_STATE_RESOLVE      "Risoluzione host remoto"
+    IDS_NFO_OVPN_STATE_TCP_CONNECT  "Connessione TCP"
+    IDS_NFO_OVPN_STATE_AUTH_PENDING "Autenticazione in attesa"
```

Comment:
maybe "In attesa di autenticazione" is more correct? Because it is not the authentication that is waiting for something, but it is the process that is waiting for the authentication to happen/finish.

Reply:
OK for "In attesa di autenticazioen".